pol-core/bscript/compiler/astbuilder/SourceFileProcessor.h

```diff
@@ -0,0 +1,41 @@
+#ifndef POLSERVER_SOURCEFILEPROCESSOR_H
+#define POLSERVER_SOURCEFILEPROCESSOR_H
+
+#include <EscriptGrammar/EscriptParserBaseVisitor.h>
+
+#include <boost/optional.hpp>
+#include <memory>
+#include <string>
+
+#include "ProgramBuilder.h"
```

Comment:
If you want to keep the "no file from the same folder unless .h/.cpp pair"
```suggestion
#include "compiler/astbuilder/ProgramBuilder.h"
```